README.md

```diff
@@ -27,6 +27,12 @@ To install the scanner globally and be able to run analyses on the command line:
 npm install -g sonarqube-scanner
 ```
 
+To run analyses without installing the scanner at all, run the following command instead:
```

Comment:
```suggestion
To run analyses without explicitly installing the scanner, run the following command instead:
```

Reply:
I'm slightly uncomfortable having this under `Installation`, would it fit better under `Usage` or somewhere else?